### PR TITLE
Not an error

### PR DESCRIPTION
**Pull request by dvertx**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**
1. Reopening debugged file with `ood` prints out an error of *ERROR: File xxx reopened in read-write mode*, instead of written as an info with RZ_LOG_INFO:
```shell
$ rizin /bin/ls
 -- Find expanded AES keys in memory with '/ca'
[0x00006ab0]> ood
ERROR: rz-run: invalid hexpair string `0`
Process with PID 5677 started...
ERROR: File dbg:///usr/bin/ls reopened in read-write mode
[0x7f94745a72b0]>
```

2. Running rizin with `-d` **always** outputs an error message of *invalid hexpair string*, also when reopened with `ood` (see number 1 above):
```shell
$ rizin -d /bin/ls
ERROR: rz-run: invalid hexpair string `0`
Process with PID 5606 started...
 -- Temporally drop the verbosity prefixing the commands with ':'
[0x7f46f789a2b0]>
```
My understanding is that `resolve_value()` gets called many times iteratively by other functions, therefore will **always** reach a `src_len`  value of 1.

...  

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/cfile.c [hunk 1]
```diff
@@ -424,7 +424,7 @@ RZ_API bool rz_core_file_reopen(RzCore *core, const char *args, int perm, int lo
 		ofile = NULL;
 		odesc = NULL;
 		//	core->file = file;
-		RZ_LOG_ERROR("File %s reopened in %s mode\n", path,
+		RZ_LOG_INFO("File %s reopened in %s mode\n", path,
```
Comment (wargio) [inline librz/core/cfile.c]: probably this should be a warning. also delete the comment above this line.

File: librz/socket/run.c [hunk 2]
```diff
@@ -253,7 +253,6 @@ static char *resolve_value(const char *src, size_t *result_len) {
 			src_len = strlen(copy);
 		}
 		if (src_len < 2 || (src_len & 1)) {
-			RZ_LOG_ERROR("rz-run: invalid hexpair string `%s`\n", copy);
```
Comment (wargio) [inline librz/socket/run.c]: Leave this as an error please.

**Review by wargio — CHANGES_REQUESTED** [review]
.

**Review by wargio — APPROVED** [review]

**Review by XVilka — CHANGES_REQUESTED** [review]
Fix the test please;
```
 [XX] db/archos/linux-x64/dbg_oo ood check for ptrace errors
RZ_NOPLUGINS=1 /usr/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -eflirt.sigdb.load.system=false -eflirt.sigdb.load.home=false -N -e log.level=5 -Qc 'ood
' bins/elf/analysis/x86-helloworld-gcc
-- stderr
--- expected
+++ actual
@@ -1,1 +1,1 @@
-Process with PID File -helloworld-gcc reopened in read-write mode
+Process with PID 
```

**Comment by dvertx** [comment]
Done. All checks now passes

**Review by wargio — APPROVED** [review]

**Review by ret2libc — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**